enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,302 @@
+
+#### Load Balancing
+When it came to load balancer, the best option we came up with was to use the IBM Cloud [VPC
+Loadbalancer][vpc-loadbalancer-website] using the Direct Link, since it allows us to provide an application load
+balancer to the cluster that can talk directly to the private network created with the [IBM Cloud DNS
+Services][ibm-cloud-dns-website]. It should be noted that the VPC Loadbalancer comes at an additional cost to the user;
+however, the Direct Link service that provides high speed connection directly to the cluster's private network is
+included with a subscriptions to Power VS.
+
+#### Separate IBM Providers for VPC vs Power VS
+One thing that has been asked and discussed is whether it is possible to create a single abstract IBM provider that
+works with IBM VPC and Power VS. Unfortunately, there are very few similiarities between the target environments outside
+of authentication. The APIs, parameters, and image management are all completely different, so we believe the cleanest
+implementation would be a separate provider.
+
+### Risks and Mitigations
+
+The Multi-Arch team has already mitigated much of the risks involved with coordinating a new ppc64le platform. We have
+existing engagements with many of the teams that will would be working with to add this platform. For instance, we
+already have a hardware environment capable of buiding the OVA images, a dedicated CI team to build out and analyze the
+development focused testing, and the underlying content release content we'd need build for ppc64le by the ART team.
+
+## Design Details
+
+The Power Systems Virtual Servers (Power VS) are separated from the rest of the IBM Cloud servers with separate networks
+and direct-attached storage. The internal networks are fenced but offer connectivity options to IBM Cloud infrastructure
+or on-premises environments. Both public and private network is supported for instances (VMs). For public networks only
+limited ports are opened for inbound access and this is not configurable.
+
+Basic Walkthrough:
+1. Create a paid IBM Cloud Account (PowerVS - or any infrastructure service, is not available for free accounts).
+2. Create Power VS service in specific region
+3. Create private network in the Power VS service
+4. [Upload/Create<sup>*</sup>](#coreos-ova-considerations) the required boot images.
+5. Create VM by specifying configurable values:
+    - \# of CPU / Cores
+    - Shared vs. Dedicated CPUs
+    - [Machine type (s922 vs. e980)](#power-cloud-instances)
+    - \# of Memory
+    - Boot Image [set in installer binary]
+    - Network (public, private) [optional]
+    - Additional disk volume(s)
+    - Storage Type (Tier 1 or 3)
+
+*Note*: The `Disk capacity` property of the image as defined in the OVA is the size of the final boot image once
+imported. The size and capacity can be different, and we're planning to keep the size as 16G with a default capacity of
```

Comment:
while this statement is true, the size here (the 16G) refers to the size of the raw disk file inside the ova not the bootimage. just as an fyi

Reply:
Added this note as a clarifying statement further in the section.